ignite-base/App/Services/RehydrationServices.js

```diff
@@ -1,16 +1,23 @@
 import ReduxPersist from '../Config/ReduxPersist'
 import { AsyncStorage } from 'react-native'
 import { persistStore } from 'redux-persist'
-import Reactotron from 'reactotron-react-native'
 
 const updateReducers = (store) => {
   const reducerVersion = ReduxPersist.reducerVersion
   const config = ReduxPersist.storeConfig
 
+
   // Check to ensure latest reducer version
   AsyncStorage.getItem('reducerVersion').then((localVersion) => {
     if (localVersion !== reducerVersion) {
-      Reactotron.log(`PURGING STORE ${localVersion} 'vs.' ${reducerVersion}`)
+      console.tron.display({
+        name: 'PURGE',
+        value: `Reducer Version Change
+        Old Version: ${localVersion}
+        New Version: ${reducerVersion}`,
+        preview: 'Purging Store',
```

Comment:
Maybe even chuck the version numbers in here, so the user doesn't even need to expand the row?  Totally your call.  I wouldn't have even thought to do this.